test/integration/runner/runner_test.go

```diff
@@ -162,17 +165,43 @@ func runSingleTest(t *testing.T, dir, binaryPath, name string) error {
 	out, err := cmd.CombinedOutput()
 	if err != nil {
 		if len(out) != 0 {
-			return fmt.Errorf(string(out))
+			return fmt.Errorf(transformOutput(string(out)))
 		}
 		return err
 	}
 
 	if testing.Verbose() {
-		t.Log(string(out))
+		if len(out) < 20000 {
```

Comment:
I thought you wanted the last 20k lines - this seems to ignore anything longer than 20k.

Reply:
I can change but last is only slightly better